### PR TITLE
Add a date filter to pours.

### PR DESCRIPTION
**Pull request by manlycode**

### CODE REVIEW

File: app/controllers/api/v1/pours_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1
+    class PoursController < ApiController
+      def index
+        @pours = Pour.for_listing.
+          between_dates(start_time: start_time, end_time: end_time)
+
+          render json: @pours
```
Comment (albus522) [inline app/controllers/api/v1/pours_controller.rb]: wonky indentation

Comment (albus522) [inline app/controllers/api/v1/pours_controller.rb]: And `.to_json` needed

Comment (danielmorrison) [inline app/controllers/api/v1/pours_controller.rb]: You shoudn't need `.to_json` these days.

File: spec/models/pour_spec.rb [hunk 2]
```diff
@@ -46,4 +46,36 @@
...
+				expect(result.size).to eql(2)
+				expect(result).to include(pour1, pour2)
+
+				result = Pour.between_dates(start_time: start_datetime, end_time: 2.weeks.ago)
+
+				expect(result.size).to eql(1)
+				expect(result).to include(pour1)
+			end
+		end
+	end
```
Comment (albus522) [inline spec/models/pour_spec.rb]: What happened to the indentation on this file?

Comment (manlycode) [inline spec/models/pour_spec.rb]: Gah! Tabs.

**Pull request merged**